rm_reporting/resources/response_chasing.py

```diff
@@ -99,3 +99,75 @@ def get(collection_exercise_id, survey_id):
         response.headers["Content-Disposition"] = f"attachment; filename=response_chasing_{collection_exercise_id}.xlsx"
         response.headers["Content-type"] = "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet"
         return response
+
+
+@response_chasing_api.route('/download-social-mi/<collection_exercise_id>')
+class SocialMIDownload(Resource):
+
+    @staticmethod
+    def get(collection_exercise_id):
+        output = io.BytesIO()
+        wb = Workbook()
+        ws = wb.active
+        ws.title = "Social MI Report"
+
+        # Set headers
+        headers = {
+            "A1": "Sample Reference",
+            "B1": "Status",
+            "C1": "Status Event",
+            "D1": "Address Line 1",
+            "E1": "Address Line 2",
+            "F1": "Locality",
+            "G1": "Town Name",
+            "H1": "Postcode",
+            "I1": "Country"
+        }
+
+        for cell, header in headers.items():
+            ws[cell] = header
+            ws.column_dimensions[cell[0]].width = len(header)
+
+        engine = app.db.engine
+
+        case_status = "SELECT DISTINCT ON (cg.sampleunitref) cg.sampleunitref, cg.status, " \
+                      "CASE WHEN ce.description !~ '[0-9]' THEN '' ELSE ce.description END, " \
+                      "attributes->> 'ADDRESS_LINE1' AS address_line_1, " \
+                      "attributes->> 'ADDRESS_LINE2' AS address_line_2, " \
+                      "attributes->> 'LOCALITY' AS locality, " \
+                      "attributes->> 'TOWN_NAME' AS town_name, " \
+                      "attributes->> 'POSTCODE' AS postcode, " \
+                      "attributes->> 'COUNTRY' AS country " \
+                      "FROM casesvc.case c JOIN casesvc.casegroup cg ON c.casegroupfk = cg.casegrouppk " \
+                      "JOIN casesvc.caseevent ce ON ce.casefk = c.casepk " \
+                      "JOIN samplesvc.sampleattributes sa ON " \
+                      "CONCAT(attributes->> 'TLA','', attributes->> 'REFERENCE') = cg.sampleunitref " \
+                      f"WHERE c.sampleunittype = 'H' AND cg.collectionexerciseid = '{collection_exercise_id}' " \
```

Comment:
It would be good practice to use a query parameter here instead of just string formatting, there's examples of that pattern in the responses dashboard endpoints.